ratis-server/src/main/java/org/apache/ratis/server/impl/PeerConfiguration.java

```diff
@@ -94,6 +94,16 @@ boolean hasMajority(Collection<RaftPeerId> others, RaftPeerId selfId) {
     return num > size() / 2;
   }
 
+  boolean majorityImpossibleWithout(Collection<RaftPeerId> rejected) {
```

Comment:
Do you think rename majorityImpossibleWithout to majorityRejectVote maybe better ?

Reply:
OK, firstly I assume rejected votes is not necessarily a majority, maybe half is enough. But I think majorityRejectVote is more self-explanatory even it is not very accurate.

Reply:
If you have better name, I also can accept it.